packages/testsuite/cypress/e2e/mail/test-configuration-subsystem-mail-credential-reference.cy.ts

```diff
@@ -0,0 +1,142 @@
+describe("TESTS: Configuration => Subsystems => Mail", () => {
+  const address = [
+    "subsystem",
+    "mail",
+    "mail-session",
+    "my-mail-session",
+    "server",
+    "smtp",
+  ];
+
+  const outBoundSocketBinding = {
+    name: "custom-outbound-socket-binding",
+    host: "localhost",
+    port: "15099",
```

Comment:
Is this just a test value or do we need the port to be really available?

Reply:
This is just test value.  We don't need that service available.